app/models/concerns/digital_object/xml_datastream_rendering.rb

```diff
@@ -156,6 +156,9 @@ def value_for_field_name(field_name, df_data)
         created_at
       elsif field_name == '$updated_at'
         updated_at
+      elsif field_name == '$doi'
+        # slice off the "doi:" label
+        doi[4..-1] unless doi.nil?
```

Comment:
Actually, I think this code should always return a value, even if doi is nil.  I'd recommend this instead:

`doi.present? ? doi.gsub(/^doi:/, '') : ''`